intel_extension_for_transformers/llm/runtime/graph/models/model_utils/model_types.h

```diff
@@ -64,7 +64,7 @@
 #ifdef __cplusplus
 extern "C" {
 #endif
-enum model_archs { MODEL_UNKNOWN, MODEL_LLAMA, MODEL_GPTJ, MODEL_MPT, MODEL_GPTNEOX, MODEL_STARCODER, MODEL_FALCON };
+enum model_archs { MODEL_UNKNOWN, MODEL_LLAMA, MODEL_GPTJ, MODEL_MPT, MODEL_GPTNEOX, MODEL_STARCODER, MODEL_FALCON, MODEL_CHATGLM, MODEL_CHATGLM2, MODEL_CHATGLM1};
```

Comment:
why there are three model rathe than MODEL_CHATGLM2, MODEL_CHATGLM1 ？ 
what MODEL_CHATGLM stand for ？